drools-wb-services/drools-wb-verifier/drools-wb-verifier-backend/src/test/java/org/drools/workbench/services/verifier/webworker/client/fromfile/DecisionTableAnalyzerFromFileTest.java

```diff
@@ -142,6 +142,24 @@ public void testFile3() throws
     }
 
     @Test
+    public void testLHSConflictsArePickedUpForEachFieldOfAPattern() throws
```

Comment:
This test uses decision table with different characteristic as the one in the jira has. This table has two conditions like (WHEN OK THEN 10, WHEN AA THEN 20) while the table in jira has different kind of conditions (WHEN A THEN 1, WHEN B THEN 1, WHEN C THEN 1) shouldn't be such table part of tests?

Reply:
@jomarko True (good spot)!

@Rikkola Probably does not hurt to include a test for the scenario in the JIRA.

Reply:
Yeah. Not a bad idea to make sure the table works. It is surprising how many bugs these dtable files are picking up and yet they are quite easy to add in.